modules/va_forms/app/workers/va_forms/form_reloader.rb

```diff
@@ -13,7 +14,11 @@ def perform
       Rails.logger.info('VAForms::FormReloader is being called.')
       query = File.read(Rails.root.join('modules', 'va_forms', 'config', 'graphql_query.txt'))
       body = { query: query }
-      response = connection.post('graphql', body.to_json)
+      response = connection.post do |req|
+        req.path = 'graphql'
+        req.body = body.to_json
+        req.options.timeout = 300
```

Comment:
5 minute timeout for a http request? Seems a bit high.

Reply:
@ericboehs So, my understanding on Faraday is that this would be the READ timeout, rather than OPEN (for opening the connection). Based on my CURL testing, the payload can take up to 3 minutes to return the data. The problem is that we are retrieving all the form data at once in a large JSON payload. It probably needs some optimization, but this is to fix a current issue where we are seeing timeouts and the forms are not getting updated.